src/components/NewBook.js

```diff
@@ -36,7 +36,7 @@ const AddBookForm = () => {
       <h1>ADD NEW BOOK</h1>
       <form className="book-form" onSubmit={handleSubmit}>
         <input
-          className="title-input"
+          className="book-input"
           type="text"
           name="title"
           value={userInput.title}
```

Comment:
- [ ] Please follow closely the Zeplin design so that your form looks exactly like the specified design guideline
- [ ] Please use any other heading tag apart from `<h1>` as it is semantically incorrect for you to have a `<h1>` that low on the page
- [ ] Please use the appropriate colors for the text, input fields and buttons 👍 
### Zeplin design
![image](https://user-images.githubusercontent.com/46839548/218285291-9d93aad7-181d-4d34-88bd-67de7e975c9e.png)
### Your design
![image](https://user-images.githubusercontent.com/46839548/218285295-23197205-782f-4708-85cc-7dd82f1e5004.png)